utilities/output_tools.py

```diff
@@ -87,8 +87,6 @@ def write_analysis_output(results, outfile, args, update_name=True):
         to_append.append(args.theoryCorr[0]+"Corr")
     if hasattr(args, "uncertainty_hist") and args.uncertainty_hist != "nominal":
         to_append.append(args.uncertainty_hist)
-    if args.postfix:
-        to_append.append(args.postfix)
     if args.correlatedNonClosureNP:
         to_append.append("NonClosureCorl")
```

Comment:
I noticed this appearing in the gen output at some point, is it the default? I think we should avoid appending too many things to the name. Just commenting here because you're touching this anyway

Reply:
I also noticed, but it doesn't come from my side. We could make a PR dedicated to "cleaning up things"

Reply:
yes fair